front/src/reducers/editor/__tests__/editorReducer.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { EditorState } from 'applications/editor/tools/types';
+import { createStoreWithoutMiddleware } from 'Store';
+import editorTestDataBuilder from './editorTestDataBuilder';
+import {
+  editorInitialState,
+  selectLayers,
+  editorSlice,
+  loadDataModelAction,
+  updateTotalsIssueAction,
+  updateFiltersIssueAction,
+} from '..';
+
+const createStore = (initialStateExtra?: EditorState) =>
+  createStoreWithoutMiddleware({
+    [editorSlice.name]: initialStateExtra,
+  });
+
+describe('editorReducer', () => {
+  const testDataBuilder = editorTestDataBuilder();
+  it('should return initial state', () => {
+    const store = createStore();
```

Comment:
Since we don't create specific store, can we create it in a beforeEach() function ?

Reply:
Done